src/Nethermind/Nethermind.Network/Config/NetworkConfig.cs

```diff
@@ -40,5 +40,7 @@ public class NetworkConfig : INetworkConfig
         public int ProcessingThreadCount { get; set; } = 1;
         public string? ClientIdMatcher { get; set; } = null;
         public bool DisableDiscV4DnsFeeder { get; set; } = false;
+        public int ThrottlingByteLimit { get; set; } = 0;
+        public int ThrottlingDelayMs { get; set; } = 0;
```

Comment:
Should we have separate configs for sending and accepting messages?

Reply:
Yes, I think this would be the better approach. Having the same config for both scenarios is confusing.

Reply:
I've separated the options for sending and receiving messages. To make things easier, options now only provide a byte limit and use a 1 second interval; that is, you can configure the number of bytes per second. In case higher precision is needed we could change it to bytes per 0.1 second (100 ms).